src/Core/RuleHelpLinkProvider.cs

```diff
@@ -31,7 +31,8 @@ public class RuleHelpLinkProvider : IRuleHelpLinkProvider
     {
         private static readonly IDictionary<string, string> repoKeyToFolderNameMap = new Dictionary<string, string>
         {
-            { SonarRuleRepoKeys.CSharpSecurityRules, "csharp" }
+            { SonarRuleRepoKeys.CSharpSecurityRules, "csharp" },
+            { SonarRuleRepoKeys.CSharpHotspotSecurityRules, "csharp" }
```

Comment:
I took the time to see if there are other hotspot/taint repositories. There are for JS/TS: https://github.com/SonarSource/sonarlint-core/blob/bb9077fbfaaaf2bb51803ceee5c3bb89adf9c607/core/src/main/java/org/sonarsource/sonarlint/core/serverapi/issue/IssueApi.java#L52

However we don't have connected mode for JS/TS, so hotspots/taint are not supported anyway. I've added this info to the connected mode ticket so we won't forget.